### PR TITLE
feat(Keyboard): add KeyboardSearch component variant

### PR DESCRIPTION
**Pull request by ladisays**
## Description

A new Keyboard variant has been created that has a keyboard format containing only letters (including the space & delete keys) and numbers.

## References

N/A

## Testing

N/A

## Automation

N/A

## Checklist

- [x] all commented code has been removed
- [x] any new console issues have been resolved
- [x] code linter and formatter has been run
- [x] test coverage meets repo requirements
- [x] PR name matches the expected semantic-commit syntax

### CODE REVIEW

File: packages/@lightningjs/ui-components/src/components/Keyboard/KeyboardSearch.stories.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import lng from '@lightningjs/core';
+import mdx from './Keyboard.mdx';
+import { CATEGORIES } from '../../docs/constants';
+import { context } from '../../globals';
+import utils from '../../utils';
+import { default as KeyboardSearchComponent } from './KeyboardSearch';
+import { Keyboard } from './Keyboard.stories';
+
+export default {
+  title: `${CATEGORIES[8]}/KeyboardSearch`,
```
Comment (ImCoolNowRight) [inline packages/@lightningjs/ui-components/src/components/Keyboard/KeyboardSearch.stories.js]: This title will need to be `'Components/Keyboard/KeyboardSearch'` now that we've upgraded to Storybook 7.

File: packages/@lightningjs/ui-components/src/components/Keyboard/KeyboardSearch.stories.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import lng from '@lightningjs/core';
+import mdx from './Keyboard.mdx';
```
Comment (ImCoolNowRight) [inline packages/@lightningjs/ui-components/src/components/Keyboard/KeyboardSearch.stories.js]: Not needed in Storybook 7.

File: packages/@lightningjs/ui-components/src/components/Keyboard/KeyboardSearch.stories.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+import utils from '../../utils';
+import { default as KeyboardSearchComponent } from './KeyboardSearch';
+import { Keyboard } from './Keyboard.stories';
+
+export default {
+  title: `${CATEGORIES[8]}/KeyboardSearch`,
+  parameters: {
+    docs: {
+      page: mdx
+    }
```
Comment (ImCoolNowRight) [inline packages/@lightningjs/ui-components/src/components/Keyboard/KeyboardSearch.stories.js]: Not needed in Storybook 7.

**Review by ImCoolNowRight — CHANGES_REQUESTED** [review]

File: packages/@lightningjs/ui-components/src/components/Keyboard/KeyboardSearch.stories.js (hunk 1 above)
Comment (ladisays) [reply packages/@lightningjs/ui-components/src/components/Keyboard/KeyboardSearch.stories.js]: Done

File: packages/@lightningjs/ui-components/src/components/Keyboard/KeyboardSearch.stories.js (hunk 2 above)
Comment (ladisays) [reply packages/@lightningjs/ui-components/src/components/Keyboard/KeyboardSearch.stories.js]: Done

File: packages/@lightningjs/ui-components/src/components/Keyboard/KeyboardSearch.stories.js (hunk 3 above)
Comment (ladisays) [reply packages/@lightningjs/ui-components/src/components/Keyboard/KeyboardSearch.stories.js]: Done

File: packages/@lightningjs/ui-components/src/components/Keyboard/KeyboardSearch.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+        ['A', 'B', 'C', 'D', 'E', 'F'],
+        ['G', 'H', 'I', 'J', 'K', 'L'],
+        ['M', 'N', 'O', 'P', 'Q', 'R'],
+        ['S', 'T', 'U', 'V', 'W', 'X'],
+        ['Y', 'Z', '1', '2', '3', '4'],
+        ['5', '6', '7', '8', '9', '0'],
+        [
+          {
+            title: 'Space',
+            size: 'lg',
```
Comment (ImCoolNowRight) [inline packages/@lightningjs/ui-components/src/components/Keyboard/KeyboardSearch.js]: I believe we'll want this to be `xl` now?

**Review by ImCoolNowRight — COMMENTED** [review]
Looking good! I think it just needs an updated width for the space bar.

File: packages/@lightningjs/ui-components/src/components/Keyboard/KeyboardSearch.js (hunk 4 above)
Comment (ladisays) [reply packages/@lightningjs/ui-components/src/components/Keyboard/KeyboardSearch.js]: I have updated the value to `xl`, thank you.

**Review by ImCoolNowRight — APPROVED** [review]

**Review by THoj13 — APPROVED** [review]

**Pull request merged**